packages/core/src/index.ts

```diff
@@ -173,6 +175,7 @@ class Hopr extends EventEmitter {
     this.environment = options.environment
     log(`using environment: ${this.environment.id}`)
     this.indexer = this.connector.indexer // TODO temporary
+    this.pubKey = new PublicKey(id.pubKey.marshal())
```

Comment:
better use `PublicKey.fromPeerId()` since `peer-id` implementation tends to change every now and then

Reply:
ok done